index.html

```diff
@@ -255,6 +255,62 @@
 
 ---
 
+# Continuous Analysis
+
+The Status Quo Doesn't Work:
+.task[
```

Comment:
I thought this was a presentation, didn't realize there was a pre-meeting component. I can do the issue post and/or rethink whether it makes sense to try to include here.